frontend/src/modules/Intersection/view/view.tsx

```diff
@@ -72,30 +72,29 @@ export function View(props: ModuleViewProps<Interfaces>): React.ReactNode {
             for (const layer of layers) {
                 if (isWellpicksLayer(layer)) {
                     layer.maybeUpdateSettings({
-                        ensembleIdent,
+                        fieldIdentifier,
                         wellboreUuid: intersectionType === IntersectionType.WELLBORE ? wellbore?.uuid : null,
                     });
                     layer.maybeRefetchData();
                 }
             }
         },
-        [layers, wellbore, ensembleIdent, intersectionType]
+        [layers, wellbore, fieldIdentifier, intersectionType]
     );
 
     React.useEffect(
         function handleTitleChange() {
-            let ensembleName = "";
-            if (ensembleIdent) {
-                const ensemble = ensembleSet.findEnsemble(ensembleIdent);
-                ensembleName = ensemble?.getDisplayName() ?? "";
+            let fieldName = "";
+            if (fieldIdentifier) {
+                fieldName = fieldIdentifier;
```

Comment:
Could optionally be further shortened to `const fieldName = fieldIdentifier ?? ""`